test/controllers/users_controller.js

```diff
@@ -0,0 +1,93 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../../app'),
+  should = chai.should(),
+  User = require('../../app/models').user;
+
+describe('users controller', () => {
+  describe('/users POST', () => {
+    context('when information is valid', () => {
+      it('should return created status', done => {
+        chai
+          .request(server)
+          .post('/users')
+          .send({
+            user: {
+              password: '1234567890',
+              email: 'federico.dinucci@wolox.com.ar',
+              firstName: 'Federico',
+              lastName: 'Di Nucci'
+            }
+          })
+          .then(res => {
+            res.should.have.status(201);
```

Comment:
you are not really validating with the model if the user was actually created, someone (not so smart) could delete your function and replace it with `res.sendStatus(201)` and it would still pass.
Here you can import the model from the same folder you use it in the project and do a findOne to check if the user was actually created.